tools/src/main/java/org/apache/kafka/trogdor/coordinator/Coordinator.java

```diff
@@ -91,7 +95,16 @@ public CoordinatorStatusResponse status() throws Exception {
     }
 
     public void createTask(CreateTaskRequest request) throws Throwable {
-        taskManager.createTask(request.id(), request.spec());
+        taskManager.createAndScheduleTask(new TaskManager.TaskDetail(request.id(), request.spec()));
+    }
+
+    public void createMultipleTasks(CreateMultipleTasksRequest request)
```

Comment:
We can simplify this by just having a single function, `void createTasks(Map<String, TaskSpec> tasks)`